sci-biology/spm/ChangeLog

```diff
@@ -0,0 +1,7 @@
+# ChangeLog for sci-biology/spm
+# Copyright 1999-2015 Gentoo Foundation; Distributed under the GPL v2
+# $Header: $
+
+  17 Feb 2015;  <chymera@gentoo.org> +metadata.xml, +spm-8.ebuild.orig,
```

Comment:
For some reason your name doesn't show and your email seems to be wrong!
```
export ECHANGELOG_USER="First Last <123@456.org>"
```
might help!